datacenterlight/views.py

```diff
@@ -106,12 +107,30 @@ def post(self, request):
         storage = request.POST.get('storage')
         storage_field = forms.IntegerField(validators=[self.validate_storage])
         template_id = int(request.POST.get('config'))
+        pricing_name = request.POST.get('pricing_name')
+        vm_pricing = VMPricing.get_vm_pricing_by_name(pricing_name)
```

Comment:
if taking `vm_pricing` from the `POST` parameters, I think we need to have some validation here so that any requests with altered `pricing_name` can be rejected.

Reply:
altered `pricing_name` as in ? For a non-existing `pricing_name`, the `get_vm_pricing_by_name` returns the default pricing. Not sure if your question was about a similar validation.

Reply:
I meant altered in a malicious way. For example, if we have different pricing on different domains, a user can get the `pricing_name` from the two pages, and then alter the `POST` data to get the same VM running at a different (cheaper) cost.

Reply:
https://redmine.ungleich.ch/issues/4496